src/trivia/util.h

```diff
@@ -573,6 +573,20 @@ int clock_gettime(uint32_t clock_id, struct timespec *tp);
 #define CLOCK_THREAD_CPUTIME_ID		3
 #endif
 
+/**
+ * Returns the escaped string if the given character should be escaped when
+ * encoded in JSON or NULL otherwise.
+ */
+static inline const char *
+json_escape_char(char c)
+{
+	extern const char *const json_char2escape[];
+	extern bool json_escape_forward_slash;
+	if (c == '/' && json_escape_forward_slash)
+		return "\\/";
+	return json_char2escape[(unsigned char)c];
```

Comment:
Since we're going to drop this change after two major releases anyway, I think it's better to keep it simple, and one condition check looks simpler to me than an extra escape table. (Let's assume we need to unescape one more character. Then we'd have to add two more tables!)